src/content/1.7/scale/optimizations.md

```diff
@@ -260,3 +260,35 @@ if (!defined('_PS_DEBUG_PROFILING_')) {
     define('_PS_DEBUG_PROFILING_', false);
 }
 ```
+
+### 8) PrestaShop database cleaning
+
+Some SQL tables of PrestaShop can grow very big in time. Some of the data being stored there is sometimes not relevant anymore or not needed anymore.
+
+Some notable examples:
+
+#### Logs table
+
+You can regularly clean your _log_ SQL table from old logs.
+
+#### Configuration table
+
+The _configuration_ SQL table can grow as you install more and more modules and a lot of them do store data inside it. Some of them do remove this data when they are being uninstalled and other do not. This can result in a _configuration_ SQL table quite big and that can be cleaned up.
```

Comment:
```suggestion
The _configuration_ table can grow as you install more and more modules and a lot of them do store data inside it. Some of them do remove this data when they are being uninstalled and other do not. This can result in a _configuration_ SQL table quite big and that can be cleaned up.
```

Reply:
Are you sure it's better this way 🤔  ? I thought that the more accurate the better ?  For beginner-level readers (that are in the process of learning PrestaShop) it might not be obvious that the configuration table is a SQL table.

Reply:
@matks I updated. After discussion with @PierreRambaud, `database table` is better.

Reply:
I'll trust you both then 👍